### ISSUE TITLE
Problem with cr_setup() - Error in if (file.exists(local_file)) { : argument is of length zero

### ISSUE THREAD

**Issue by Camilonfc** [MarkEdmondson1234/googleCloudRunner#120]
Hello Mark, I hope you are well.
I write since I am having problems trying to configure the package (googleCloudRun), I was following the configuration video, but from the beginning I have problems. I tried downloading the package from CRAN and your dev version, and I have the same problem, it happens that once the package is installed and I execute the statement library (googleCloudRun)> cr_setup () the configuration menu opens, but when pressing any of The options gives me the following error: 
(Error in if (file.exists (local_file)) {: argument is of length zero), 
probe uninstalling, restarting RStudio, reinstalling, but the problem continues, I am working with R version 4.1 .0 (2021-05-18).
I would appreciate your help to see what may be happening, since I need to automate my R scripts in GCP.
Greetings!
[console log.txt](https://github.com/MarkEdmondson1234/googleCloudRunner/files/6617276/console.log.txt)

**Comment by MarkEdmondson1234** [comment]
Could you also run `sessionInfo()` to show your system?  I'm guessing its not finding an `.Renviron` file in your home directory.  You could try creating that.  Its a bug though that it fails if it doesn't exist if that is the case.

**Comment by Camilonfc** [comment]
Thanks for your answer, I just checked, leave my information attached, anyway in the work path if I have created a .Renviron file (this was created by watching the video), I don't know if I should leave it in another path or it should go in the project work path.
[sessInfo.txt](https://github.com/MarkEdmondson1234/googleCloudRunner/files/6617344/sessInfo.txt)

**Comment by Camilonfc** [comment]
I think I found the problem, the .Renviron file in my Home directory was empty, I copied the information from the .Renviron file of my project and it worked, but with another error, but I think it is due to configuration issues.

> library (googleCloudRunner)
v Setting scopes to https://www.googleapis.com/auth/cloud-platform
x Failed! Auto-authentication via GCE_AUTH_FILE = C: \ Users \ cflandez \ SURA INVESTMENT MANAGEMENT SA \ Code - General \ SOLUCIONES \ Investment Analytics - Condor-wm \ GCP Test \ googlecloudrunner-client.json - error was: $ private_key not found in JSON - have you downloaded the correct JSON file?
(Service Account Keys, not service account client)

I will continue with the video, maybe I missed setting a parameter or I made a mistake in something.
Thanks again!

**Comment by Camilonfc** [comment]
I think I found the problem, the .Renviron file in my Home directory was empty, I copied the information from the .Renviron file of my project and it worked, but with another error, but I think it is due to configuration issues.

> library (googleCloudRunner)
v Setting scopes to https://www.googleapis.com/auth/cloud-platform
x Failed! Auto-authentication via GCE_AUTH_FILE = C: \xxxxxxxxxxxxxxxx\ GCP Test \ googlecloudrunner-client.json - error was: $ private_key not found in JSON - have you downloaded the correct JSON file?
(Service Account Keys, not service account client)

I will continue with the video, maybe I missed setting a parameter or I made a mistake in something.
Thanks again
